machine/install-machine.md

```diff
@@ -80,7 +80,7 @@ done
 ```
 
 Then you need to run `source
-/etc/bash_completion.d/docker-machine-prompt.bash` in your bash
+/etc/bash_completion.d docker-machine-prompt.bash` in your bash
```

Comment:
Closing ticket for lack of response. If you feel this is in error, feel free to reopen. Thanks!